### PR TITLE
fix: feed settings survey border tweaks

### PR DESCRIPTION
**Pull request by omBratteng**
I tweaked it a little bit

#3174 

<table>
<tr>
<td>Before
<td>After
<tr>
<td>
<td>
</table>

### Preview domain
https://mi-355-tweaks.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/cards/survey/FeedSurveyCard.tsx [hunk 1]
```diff
@@ -21,21 +21,21 @@ export function FeedSurveyCard({
...
       style={{
         backgroundImage:
           'linear-gradient(180deg, rgba(255, 233, 35, 1) 0%, rgba(252, 83, 141, 1) 50%, rgba(113, 71, 237, 1) 100%)',
       }}
     >
-      <div className="absolute inset-px rounded-16 bg-background-default" />
       <CardComponent
         className={classNames(
-          'relative items-center gap-4 overflow-hidden',
+          '!bg-background-default/[.92] shadow-none relative items-center gap-4 overflow-hidden !rounded-14 border-0',
```
Comment (omBratteng) [inline packages/shared/src/components/cards/survey/FeedSurveyCard.tsx:32]: `!bg-background-default/[.92] shadow-none` should've worked according to Tailwind docs, but they didn't locally. So I added their equivalent in `style`

File: packages/shared/src/components/cards/survey/FeedSurveyCard.tsx [hunk 2]
```diff
@@ -21,21 +21,21 @@ export function FeedSurveyCard({
 
   return (
     <div
-      className="relative rounded-16 p-px"
+      className="relative rounded-16 p-0.5"
```
Comment (omBratteng) [inline packages/shared/src/components/cards/survey/FeedSurveyCard.tsx:24]: Added an extra pixel on padding to make the gradient border a bit more clear

File: packages/shared/src/components/cards/survey/FeedSurveyCard.tsx (hunk 1 above)
Comment (nensidosari) [reply packages/shared/src/components/cards/survey/FeedSurveyCard.tsx:32]: Do we still need to keep this here since it doesn't work?

Comment (omBratteng) [reply packages/shared/src/components/cards/survey/FeedSurveyCard.tsx:32]: I removed it. And fixed so `shadow-none` works

**Review by nensidosari — APPROVED** [review]

**Review by rebelchris — APPROVED** [review]

File: packages/shared/src/components/cards/survey/FeedSurveyCard.tsx [hunk 3]
```diff
@@ -22,13 +22,12 @@ export function FeedSurveyCard({
...
       style={{ backgroundImage: feedSurveyBorder }}
     >
-      <div className="absolute inset-px rounded-16 bg-background-default" />
       <CardComponent
         className={classNames(
-          'relative items-center gap-4 overflow-hidden',
+          'relative items-center gap-4 overflow-hidden !rounded-14 border-0 shadow-none',
```
Comment (sshanzel) [inline packages/shared/src/components/cards/survey/FeedSurveyCard.tsx:30]: I think we don't have to override the roundness anymore as it is naturally found at the `CardComponent`.
```suggestion
          'relative items-center gap-4 overflow-hidden border-0 shadow-none',
```

**Review by sshanzel — APPROVED** [review]
Looks amazing to me. Thank you so much for the effort 🙇

File: packages/shared/src/components/cards/survey/FeedSurveyCard.tsx (hunk 3 above)
Comment (omBratteng) [reply packages/shared/src/components/cards/survey/FeedSurveyCard.tsx:30]: We still need it, as `CardComponent` has a too rounded radius.
<table>
<tr>
<td>rounded-16
<td>!rounded-14
<tr>
<td>
<td>

Comment (omBratteng) [reply packages/shared/src/components/cards/survey/FeedSurveyCard.tsx:30]: `inner-radius = outer-radius - padding`
I.e. 16px (outer radius) - 2px (padding) = 14px

**Pull request merged**